base/float.jl

```diff
@@ -400,9 +400,19 @@ end
 isequal(x::Float16, y::Float16) = fpiseq(x, y)
 isequal(x::Float32, y::Float32) = fpiseq(x, y)
 isequal(x::Float64, y::Float64) = fpiseq(x, y)
-isless( x::Float16, y::Float16) = fpislt(x, y)
-isless( x::Float32, y::Float32) = fpislt(x, y)
-isless( x::Float64, y::Float64) = fpislt(x, y)
+
+# interpret as sign-magnitude integer
+@inline function _fpint(x)
+    IntT = inttype(typeof(x))
```

Comment:
probably better to just use  the following and to avoid having to define new  float to int  methods
```suggestion
    IntT = signed(uinttype(typeof(x)))
```

Reply:
Thanks, fixed!